src/attributes/classes.js

```diff
@@ -158,12 +156,13 @@ jQuery.fn.extend({
 	},
 
 	hasClass: function( selector ) {
-		var className = " " + selector + " ",
-			i = 0,
-			l = this.length;
-		for ( ; i < l; i++ ) {
-			if ( this[i].nodeType === 1 &&
-				( " " + getClass( this[i] ) + " " ).replace( rclass, " " )
+		var className, elem,
+			i = 0;
+
+		className = " " + selector + " ";
```

Comment:
If I do that `jscs` (after doing `$> grunt`) tell me:
```
Var declarations should be joined at src/attributes/classes.js :
   160 |   i = 0;
   161 |
   162 |  var className = " " + selector + " ";
----------^
   163 |  while ( ( elem = this[ i++ ] ) ) {
   164 |   if ( elem.nodeType === 1 &&
>> 1 code style errors found!
Warning: Task "jscs:src" failed. Use --force to continue.
```

But if I do this:
``` javascript
var elem,
	className = " " + selector + " ",
	i = 0;
```
it cost +4 bytes (total -22)

and if I do this (more simple):
``` javascript
var elem,
	i = 0;
selector = " " + selector + " ";
```
it cost +5 bytes (total -21)